eng/common/LiveTestResources/New-TestResources.ps1.md

```diff
@@ -0,0 +1,392 @@
+---
+external help file: -help.xml
+Module Name:
+online version:
+schema: 2.0.0
+---
+
+# New-TestResources.ps1
+
+## SYNOPSIS
+Deploys live test resources defined for a service directory to Azure.
+
+## SYNTAX
+
+### Default (Default)
+```
+New-TestResources.ps1 [-BaseName] <String> -ServiceDirectory <String> -TestApplicationId <String>
+ [-TestApplicationSecret <String>] [-TestApplicationOid <String>] [-DeleteAfterHours <Int32>]
+ [-Location <String>] [-AdditionalParameters <Hashtable>] [-CI] [-Force] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### Provisioner
+```
+New-TestResources.ps1 [-BaseName] <String> -ServiceDirectory <String> -TestApplicationId <String>
+ [-TestApplicationSecret <String>] [-TestApplicationOid <String>] -TenantId <String>
+ -ProvisionerApplicationId <String> -ProvisionerApplicationSecret <String> [-DeleteAfterHours <Int32>]
+ [-Location <String>] [-AdditionalParameters <Hashtable>] [-CI] [-Force] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+If a service directory contains one or more ARM templates named
```

Comment:
How can a directory contain more than one test-resources.json?

Reply:
Thanks for catching that. I updated the wording.

This documentation was largely unchanged from when it was originally authored (https://github.com/Azure/azure-sdk-for-net/commit/f21f587269f370e8c0f506183045d8dfc521085b#diff-c83125d763bfeb5bef5fe5f1363b6ac2R256)